docs/ui.md

```diff
@@ -1,20 +1,43 @@
 # Kubernetes UI Instructions
 
 ## Kubernetes User Interface
-Kubernetes has an extensible user interface with default functionality that describes the current cluster. See the [README](../www/README.md) in the www directory for more information.
+Kubernetes has a web-based user interface that displays the current cluster state graphically. 
 
-### Running locally
-Assuming that you have a cluster running locally at `localhost:8080`, as described [here](getting-started-guides/locally.md), you can run the UI against it with kubectl:
+## Accessing the UI
+By default, the Kubernetes UI is deployed remotely as a cluster addon. To access it, visit `/ui` on your master server, which redirects to `/api/v1/proxy/namespaces/kube-system/services/kube-ui/#/dashboard/`.
 
-```sh
-kubectl proxy --www=www/app --www-prefix=/
+If the UI cluster addon isn't running, you can start it manually with the following commands.
+```shell
+cd cluster/addons/kube-ui/image
```

Comment:
These are commands for building the ui. To just start the UI, it should be this:

```
$ kubectl create -f cluster/addons/kube-ui/kube-ui-svc.yaml
$ kubectl create -f cluster/addons/kube-ui/kube-ui-rc.yaml
```